addons/main/script_macros_common.hpp

```diff
@@ -518,6 +539,27 @@ Macro: DEC()
 #define DEC(var) var = (var) - 1
 
 /* -------------------------------------------
+Macro: INV()
+
+Description:
+    Invert a number's sign. Opposite of the abs command with positive numbers.
+
+Parameters:
+    VAR - Variable to invert [Number]
+
+Example:
+    (begin example)
+    _degrees = 45;
+    INV(_degrees);
+    // _degrees => -45
+    (end)
+
+Author:
+    654wak654
+------------------------------------------- */
+#define INV(var) var = (var) * -1
```

Comment:
That... Makes a lot of sense actually. Do you know what happens if `_var` is already negative?